blog/settings.py

```diff
@@ -2,7 +2,7 @@
 
 import environ
 
-env = environ.Env(DEBUG=(bool, False),)
+env = environ.Env(DEBUG=(bool, True),)
```

Comment:
Please revert this change. defaults are better to be production-ready. add `DEBUG=on` to `.env` instead.